### PR TITLE
chore: Add integration test for Python stack

### PR DESCRIPTION
**Pull request by lym953**
### What does this PR do?
Add a Python stack to integration tests, so that the test will compare the snapshots generated after vs before, to check if a PR causes unexpected changes on the stack to be deployed.

The stack was copied from `examples/python-stack`. I had to copy it because I was unable to find a way to just reference the existing stack.

### Motivation
Adding more tests to make it safer to deploy changes.

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

File: integration_tests/snapshots/correct-lambda_python_stack-snapshot.json
Comment (purple4reina) [inline integration_tests/snapshots/correct-lambda_python_stack-snapshot.json:1]: Is this file suppose to be empty?

Comment (purple4reina) [inline integration_tests/snapshots/correct-lambda_python_stack-snapshot.json:1]: Is this file suppose to be empty?

Comment (lym953) [reply integration_tests/snapshots/correct-lambda_python_stack-snapshot.json:1]: Good catch! Fixed.

Comment (lym953) [reply integration_tests/snapshots/correct-lambda_python_stack-snapshot.json:1]: Good catch! Fixed.

Comment (lym953) [reply integration_tests/snapshots/correct-lambda_python_stack-snapshot.json:1]: Done!

File: integration_tests/snapshots/test-lambda_python_stack-snapshot.json
Comment (purple4reina) [inline integration_tests/snapshots/test-lambda_python_stack-snapshot.json:1]: Does it seem weird to you that these `integration_tests/snapshots/test-*` files are committed to git? Seems like they're overwritten by the test runner anyway.

**Review by purple4reina — APPROVED** [review]
There's still an integration test failure, but other than that this looks great!

File: integration_tests/snapshots/test-lambda_python_stack-snapshot.json
Comment (lym953) [reply integration_tests/snapshots/test-lambda_python_stack-snapshot.json:1]: Good point. I'm already working on that. Coming soon in the next PR!

**Pull request merged**